echo-pubsub-core/src/main/java/com/netflix/spinnaker/echo/pubsub/CDEventController.java

```diff
@@ -154,7 +160,7 @@ public ResponseEntity<Void> produceEvent() throws IOException, TimeoutException
     data.setSubject("cdevent");
     objectMapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
     CloudEvent cloudEvent =
-        cdEventCreator.createArtifactEvent(
+        dev.cdevents.CDEventTypes.createArtifactEvent(
```

Comment:
you can avoid package name here and all the places below, just `import dev.cdevents.CDEventTypes`